specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -4734,6 +4734,9 @@
           },
           "Create a scale set with userData.": {
             "$ref": "./examples/CreateOrUpdateAScaleSetWithUserData.json"
+          },
+          "Create a scale set with Capacity Reservation": {
+            "$ref": "./examples/CreateOrUpdateAScaleSetWithCapacityReservation.json"
```

Comment:
please put this file in examples/compute folder

Reply:
updated